mlflow/server/js/src/components/modals/InputFormView.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { Form, Input } from 'antd';
+
+export const NEW_NAME_FIELD = 'newName';
+
+/**
+ * Component that renders a form for updating a run's or experiment's name.
+ */
+class InputFormViewComponent extends Component {
```

Comment:
That was actually the first name that I chose (`RenameModal` respectively). However, I thought that the component (as well as the "generic" modal) could be used for a `Create Experiment` use case as well. If you could provide me with a mockup that includes some kind of `Create` button, I could add it to the PR. The "create experiment from UI" has also been requested by the community before.

Reply:
Thank you for adding the rename and delete features! Looks great to me.

For the new experiment button, see a mock below. I use a 13px Font awesome **fa-plus** with the following styles `width: 24px;
  height: 24px;
  color: #333333;
  background-color: #f5f5f5;
  border-color: #cccccc;
  border-radius: 4px;
  margin-right: 12px;
`  

![MLFlow_NewExperiment](https://user-images.githubusercontent.com/1960721/73681541-67825700-4673-11ea-93fb-8e94c70d0483.png)

Reply:
Thanks for providing the mockup. I added the functionality and updated the PR description.
CC @Zangr for the code review.

Reply:
Thank you for adding this so quickly! Btw shall we direct users to the newly created experiment page after the creation dialog? So they don't need to manually find the new experiment in the list.

Reply:
Sure, we could do that. I also thought about it while implementing the feature but wasn't sure about it since no runs will be shown until executing a run & refreshing the page. 
Would you like me to change it?

Reply:
Yes! I assume the next natural step for users is to copy the experiment ID to log runs. So it would be nice to show the new experiment.

Reply:
I totally agree! The redirect should work now.